docs/concepts/what-is-an-event.md

```diff
@@ -7,65 +7,144 @@ tags:
   - explanation
 ---
 
-Events are created by the Temporal Cluster in response to external occurrences and Commands generated by a Workflow Execution.
+Events are created by the Temporal Cluster in response to external occurrences and Commands generated by a Workflow Execution. Each Event corresponds to an `enum` that is defined in the [Server API](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/event_type.proto).
 
 All Events are recorded in the [Event History](/docs/concepts/what-is-an-event-history).
 
-The following is a complete list of all possible Events that could appear in a Workflow Execution Event History.
-
-Each Event corresponds to an `enum` that is defined in the [Server API](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/event_type.proto).
+A list of all possible Events that could appear in a Workflow Execution Event History is provided below.
 
 ### WorkflowExecutionStarted
 
-This is always the first Event in a Workflow Execution History and it indicates that the execution was invoked.
+This is always the first Event in a Workflow Execution History. It indicates that the execution was invoked.
+
+| Field | Type | Definition |
+| ----- | ---- | ---------- |
 
 ### WorkflowExecutionCompleted
 
-This event type indicates that the Workflow Execution has successfully completed and contains Workflow Execution results.
+This indicates that the Workflow Execution has successfully completed. The Event contains Workflow Execution results.
+
+| Field                            | Type     | Definition                                                                        |
```

Comment:
I am thinking that we should go with Description column name.

Definition -> Description